swagger.yaml

```diff
@@ -304,7 +304,132 @@ paths:
                   $ref: "#/components/schemas/response"      
         400:          
           description: 'failed operation'  
-
+    post:
+      tags:
+        - 'Articles'
+      summary: Create an article on errorswag
+      description: Create an article
+      operationId: CreateArticle
+      parameters:
+        - in: query
```

Comment:
Please revisit this. The create article route doesn't have a query

Reply:
Yeah, thanks for this. I didn't notice that.
This will be resolved.